grails-app/init/org/pih/warehouse/BootStrap.groovy

```diff
@@ -467,43 +470,62 @@ class BootStrap {
                 throw new RuntimeException("Connection could not be created.")
             }
 
-            /*
-             * FIXME This patch is unlikely to work in production. OBGM-333
-             *
-             * Previously, we used org.liquibase.grails.GrailsFileOpener here
-             * (from a plugin, https://github.com/liquibase/grails-liquibase/,
-             * that ceased development at Grails 1.3.6).
-             *
-             * The path manipulation here is probably dependent on a local install.
-             */
```

Comment:
most of the changes here aren't strictly new, but rather pull code from a previous rebase